client/src/components/settings/Certification.js

```diff
@@ -371,7 +371,7 @@ export class CertificationSettings extends Component {
             })}
           </p>
           <ul>
-            <li>Responsive Web Design</li>
+            <li>{t(`certs:certNames.${'Responsive Web Design'}`)}</li>
```

Comment:
If we take this approach, will need to translate the rest of the `li` elements.

Also probably switch to bracket notation instead of template string.